tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -1171,6 +1171,23 @@ def _impl(ctx):
         ],
     )
 
+    generate_linkmap_feature = feature(
+        name = "generate_linkmap",
+        flag_sets = [
+            flag_set(
+                actions = [
+                    ACTION_NAMES.cpp_link_executable,
+                ],
+                flag_groups = [
+                    flag_group(
+                        flags = ["-Wl,-Map=%{output_execpath}.map"],
```

Comment:
on apple platforms this has to be lowercase `-map` with a space instead of an `=`, so `-Wl,-map=whatever` 🙃

Reply:
I don't see the space in your example.

Reply:
ah sorry `-Wl,-map,whatever`

Reply:
https://stackoverflow.com/a/2391623 mentions using a comma instead of `=` or space.
I think that is nicer than a space.

Reply:
yea, the comma is replaced with a space in my second example, just can't be `=` since that's literal